WeScanSampleProject/HomeViewController.swift

```diff
@@ -97,6 +97,12 @@ final class HomeViewController: UIViewController {
     @objc func scanOrSelectImage(_ sender: UIButton) {
         let actionSheet = UIAlertController(title: "Would you like to scan an image or select one from your photo library?", message: nil, preferredStyle: .actionSheet)
         
+        let newAction = UIAlertAction(title: "New", style: .default) { (_) in
```

Comment:
Can we make the distinction between "scan" and "new" a bit more clear? From the user perspective of the example app, "scan" is the same as "new", as in, "a new scan"

Reply:
`A new scan` is ok 👍🏻